### ISSUE TITLE
Remove Kubernetes-api-java integration test

### ISSUE THREAD

**Issue by chbatey** [akka/akka-management#450]
There's nothing specific to Java/Scala. I believe we had it to show how to use Maven but we now have an external sample. WDYT @raboof ? It doesn't use master as it isn't part of the sbt build

**Comment by raboof** [comment]
Makes sense to remove it here as it is unlikely to break, and when it breaks we will find out when the example in the separate repository breaks :+1:
